commercetools-models/src/main/java/io/sphere/sdk/products/search/PriceSelectionBuilder.java

```diff
@@ -80,6 +86,16 @@ public PriceSelectionBuilder priceChannel(@Nullable final Referenceable<Channel>
         return this;
     }
 
+    public PriceSelectionBuilder storeProjection(@Nullable final String storeProjection) {
+        this.storeProjection = storeProjection;
+        return this;
+    }
+
+    public PriceSelectionBuilder localeProjection(@Nullable final String localeProjection) {
+        this.localeProjection = localeProjection;
+        return this;
+    }
```

Comment:
Here we would have to add a `plusLocaleProjection(@Nullable final String localeProjection)` method that adds the given `localeProjection` to the `localeProjection` member variable of this object.